lib/hope-tab.js

```diff
@@ -0,0 +1,305 @@
+        position: relative;
+        height: ${BAR_HEIGHT}px;
+
+        line-height: ${BAR_HEIGHT}px;
+        background: #56565A;
+        color: white;
+      }
+
+      .bar .tab {
+        display: flex;
+        align-items: center;
+      }
+
+      .bar .title {
+        flex: 1;
+        margin: 0;
+        padding: 0 16px;
+        overflow: hidden;
+
+        white-space: nowrap;
+        text-overflow: ellipsis;
+        font-size: 18px;
+        font-weight: normal;
+        text-shadow: 0 1px 0 rgba(0,0,0,0.15);
+        pointer-events: none;
+      }
+
+      .bar .close {
+        height: ${BAR_HEIGHT}px;
+        padding: 0 8px;
+
+        font-size: 32px;
+        transition: opacity 200ms ease-in;
+      }
+
+      .url {
+        position: absolute;
+        top: 0;
+        left: 0;
+        z-index: 500;
+
+        width: 100%;
+        height: 100%;
+        padding: 8px;
+        box-sizing: border-box;
+        overflow: hidden;
+
+        opacity: 0;
+        visibility: hidden;
+        background: #333;
+        transition:
+          opacity 250ms ease-in,
+          visibility 250ms ease-in;
+      }
+
+      [active] .url {
+        opacity: 1;
+        visibility: visible;
+      }
+
+      .url .field {
+        position: relative;
+        display: flex;
+        align-items: center;
+        height: 100%;
+        background: rgba(255,255,255,0.05);
+      }
+
+      .url .field .reader,
+      .url .field .refresh {
+        width: 16px;
+        height: 16px;
+        background-size: 100%;
+        background-position: center;
+        background-repeat: no-repeat;
+        background-clip: padding-box;
+        border: solid 12px transparent;
+        box-sizing: content-box;
+      }
+
+      .url .field .reader {
+        background-image: url(assets/urlbar_reader_mode_icon.png);
+      }
+
+      .url .field .refresh {
+        background-image: url(assets/urlbar_refresh_icon.png);
+      }
+
+      .url .url-text {
+        display: block;
+        flex: 1;
+        overflow: hidden;
+
+        white-space: nowrap;
+        text-overflow: ellipsis;
+        line-height: 52px;
+        text-align: center;
+        font-size: 18px;
+        font-weight: 400;
+        color: white;
+        opacity: 1;
+
+        transition-duration: 300ms;
+        transition-timing-function: ease-in;
+        transition-property: opacity;
+      }
+
+      .content {
+        position: absolute;
+        top: ${BAR_HEIGHT}px;
+        bottom: 0;
+        left: 0;
+        right: 0;
+      }
+
+      iframe {
+        width: 100%;
+        height: 100%;
+        border: 0;
+      }
+
+      .overlay {
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+
+        opacity: 0.5;
+        transition: opacity 250ms ease-in, visibility 250ms ease-in;
+        visibility: visible;
+      }
+
+      [active] .overlay {
+        opacity: 0;
+        pointer-events: none;
```

Comment:
this will cause a reflow when entering/leaving the tabs view. and I really think we should keep this interaction reflow free if we don't want it to get more and more costly with time.

Just setting `visibility: hidden` instead of `pointer-events: none` should be enough. (it will be toggled at the right time thanks to the transition)